doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,127 @@
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+Building for the application core
+*********************************
+
+For instructions on building for the application core only, see how to :ref:`program an application <programming_cmd>`.
+
+Building for the application and FLPR core
+******************************************
+
+This section outlines how to build and program for both the application and FLPR core, covering separate builds and sysbuild configurations.
+FLPR core supports two variants:
+
+* ``nrf54l15dk/nrf54l15/cpuflpr`` where FLPR runs from SRAM (recommended).
+   To build FLPR image with this variant, the application core image must include the ``nordic-flpr`` snippet.
+
+* ``nrf54l15dk/nrf54l15/cpuflpr/xip`` where FLPR runs from RRAM.
+   To build FLPR image with this variant, the application core image must include the ``nordic-flpr-xip`` snippet.
+
+Separate images
+---------------
+
+To build and program the application sample and the FLPR sample as separate images, follow these steps:
+
+.. tabs::
+
+   .. group-tab:: west
+
+      1. |open_terminal_window_with_environment|
+      #. Build the application core image, and based on your build target include the appropriate snippet:
+
+         .. code-block:: console
+
+            west build -p -b nrf54l15dk/nrf54l15/cpuapp -S nordic-flpr --no-sysbuild
+
+      #. Flash the application core image by running the following command:
+
+         .. code-block:: console
+
+            west flash
+
+      #. Build the FLPR core image:
+
+         .. code-block:: console
+
+            west build -p -b nrf54l15dk/nrf54l15/cpuflpr --no-sysbuild
+
+      #. Once you have successfully built the FLPR core image, flash it onto your device:
+
+         .. code-block:: console
+
+            west flash
+
+   .. group-tab:: nRF Util
+
+      1. |open_terminal_window_with_environment|
+      #. For the FLPR core, run the following command to erase the flash memory of the FLPR core and program the sample:
+
+         .. Commands TBD
+
+         .. code-block:: console
+
+            nrfutil device program --firmware zephyr.hex --options chip_erase_mode=ERASE_ALL --core Network
+
+      #. For the application core, navigate to its build folder and run:
+
+         .. code-block:: console
+
+            nrfutil device program --firmware zephyr.hex  --options chip_erase_mode=ERASE_ALL
+
+         .. note::
+            The application build folder will be in a sub-directory which is the name of the folder of the application
+
+      #. Reset the development kit:
+
+         .. Commands TBD
+
+         .. code-block:: console
+
+            nrfutil device reset --reset-kind=RESET_PIN
+
+         See :ref:`readback_protection_error` if you encounter an error.
+
+With sysbuild
+-------------
+
+To build and program with sysbuild, complete the following steps:
```

Comment:
Sysbuild is enabled by default in the NCS https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/config_and_build_system.html#sysbuild_enabled_by_default , so how come we highlight this here?

Reply:
I suggest renaming this to something along the lines of `Standard build` and putting this before `Separate images`.

Reply:
updated as suggested